tests/GcloudTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Tests;
+
+class GcloudTest extends \PHPUnit_Framework_TestCase
+{
+    public function testTests()
+    {
+        $this->assertEquals(true, true);
```

Comment:
Ah, good eye! Yes, we do need this test but it should probably actually
test something :). I'll make sure it gets knocked out.

On Fri, Mar 25, 2016, 8:24 PM Stephen Sawchuk <notifications@github.com>
wrote:

> In tests/GcloudTest.php
> <https://github.com/GoogleCloudPlatform/gcloud-php/pull/24#discussion_r57504904>
> :
>
> > + *     http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +namespace Google\Cloud\Tests;
> > +
> > +class GcloudTest extends \PHPUnit_Framework_TestCase
> > +{
> > +    public function testTests()
> > +    {
> > +        $this->assertEquals(true, true);
>
> Do you need this test?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-php/pull/24/files/58dae2d85d6bbc102dd3936b4797cf0b7a07305b#r57504904>
>